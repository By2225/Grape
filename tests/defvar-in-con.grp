fun Int main() {
    Int a = 3;
    Int b;
    if( a== 3) {
        b = 5;
    }
    Int c = 35;
    print(b);
    print(c);
}
