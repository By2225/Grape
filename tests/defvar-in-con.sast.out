
fun Int main() {
Int (Int : a = (Int : 3));
Int b;
if ((Bool : (Int : a) == (Int : 3)))
{
(Int : b = (Int : 5));
}
Int (Int : c = (Int : 35));
(* : print((Int : b)));
(* : print((Int : c)));
}
